### PR TITLE
Update workato api doc

### PR DESCRIPTION
**Pull request by eeliang**
Updating https://github.com/workato/docs/pull/900: 

New changes in #901 :
- Apply `isTocVisible` to all pages
- Update endpoints to use `blockquotes`
- Differentiating **URL parameters** and **Request body**
- Removing error code from individual pages, error codes will be shown in workato-api.md
- \<div class\> tags for api tables, see https://github.com/workato/docs/pull/903.
- Standarized all API headings at H2

### CODE REVIEW

File: workato-api/connections.md [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
 date: 2019-03-21 11:20:00 Z
+isTocVisible: true
 ---
 
 # Connections
+
 Use the following endpoints to retrieve the connections that belong to a user.
 
-## Supported Formats
+### Supported Formats
```
Comment (kandadaboggu) [inline workato-api/connections.md]: We can remove this from individual pages. We can add the format information in the main page.

File: workato-api/connections.md [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
+
 * JSON
 
 ## List Connections
 
-> GET /api/connections  
+Returns all connections and associated data for the authenticated Workato user .
```
Comment (kandadaboggu) [inline workato-api/connections.md]: Remove the space before the period.

File: workato-api.md [hunk 3]
```diff
@@ -5,24 +5,25 @@ isTocVisible: true
...
-| `200` | Success |
-| `400` | Bad request
+| Code  | Description  |
+|-------|--------------|
+| `200` | Success      |
+| `400` | Bad request  |
 | `401` | Unauthorized |
-| `404` | Not found |
+| `404` | Not found    |
 | `500` | Server error |
```
Comment (kandadaboggu) [inline workato-api.md]: Add section for reply format (i.e. JSON)
And a sample error reply

Comment (eeliang) [reply workato-api.md]: added, see line 26.

**Pull request merged**